lib/heap/index.js

```diff
@@ -25,7 +25,7 @@ var Heap = module.exports = integration('Heap')
  */
 
 Heap.prototype.initialize = function(page){
-  window.heap=window.heap||[];window.heap.load=function(a){window._heapid=a;var d=function(a){return function(){window.heap.push([a].concat(Array.prototype.slice.call(arguments,0)));};},e=["identify","track"];for (var f=0;f<e.length;f++)window.heap[e[f]]=d(e[f]);};
+  window.heap=window.heap||[];window.heap.load=function(a){window.appid=a;var d=function(a){return function(){window.heap.push([a].concat(Array.prototype.slice.call(arguments,0)));};},e=["identify","track"];for (var f=0;f<e.length;f++)window.heap[e[f]]=d(e[f]);};
```

Comment:
Looks like they've changed their snipped quite a lot with this new version - now it creates its own script tags. The [Localytics snippet](https://support.localytics.com/Web) does a very similar thing, and so we've split it out so that when we initialize, it only does the things it needs to. Localytics is [here](https://github.com/segmentio/analytics.js-integrations-private/blob/master/lib/localytics/index.js#L38-L50), and google analytics does a similar thing [here](https://github.com/segmentio/analytics.js-integrations/blob/master/lib/google-analytics/index.js#L109-L111).